project.ruleset.xml.example

```diff
@@ -14,6 +14,9 @@
 	<!-- Exclude the Composer Vendor directory. -->
 	<exclude-pattern>/vendor/*</exclude-pattern>
 
+	<!-- Exclude minified Javascript files. -->
+	<exclude-pattern>*.min.js</exclude-pattern>
+
```

Comment:
Good idea. Is `/node_modules/*` the pattern ? I have never used node, so no idea what to exclude.